openstudiocore/ruby/openstudio_cli.rb

```diff
@@ -1,3 +1,13 @@
-load "#{File.dirname(__FILE__)}/openstudio.rb"
+require "openstudio-workflow"
 
-puts 'hello world'
+osw_path = ARGV[0]
```

Comment:
@henryHorsey this is my super lame cli, you can extend this file to add real options parsing